### DISCUSSION TITLE
ubo logger filter lookup issue

### DISCUSSION THREAD

**Discussion by ghajini** [uBlockOrigin/uBlock-issues#3020] [General] [answered]
iam seeing filters lookup are cropped on **firefox android**

ublock origin 1.54
eg this filter `||userapi.com/impf/$image,uritransform=/^\/impf\/+//,domain=vk.com|vk.ru` from ublock annoyances looks in logger as below ie **filter is cropped on right edge**

![Screenshot_20231214-214512-593.png](https://github.com/uBlockOrigin/uBlock-issues/assets/20338483/62a5a170-32e7-4768-bddf-6f132aa74d41)

**Comment by u-RraaLL** [comment, marked as answer]
https://github.com/gorhill/uBlock/commit/49c8310e2257da4e249fe5fd721817a7a20c3c3d
